NEWS.md

```diff
@@ -6,7 +6,9 @@
 
 ## Breaking Changes
 
-* `renv` and related files have been removed (#360)
+* `renv` and related files have been removed. (#360)
+
+* No longer exporting `is_named()` function. (#401)
```

Comment:
I think we should follow our deprecation process.

Reply:
Thinking a bit more about the logistics here. I don't think there is a way to import the function from rlang and deprecate its exported equivalent at the same time.

I see two options:
1. Leave it as is as a breaking change, and remove the function from export. I can't imagine users relying on an exported version of `is_named()` when there is an rlang equivalent.
2. We can deprecate our implementation of `is_named()`, but we cannot import `is_named()`. We would need to change is call to the function adding the rlang namespace.

Reply:
I say we go for option 1 and just make a note of it in the news.

I did a quick search of all pharmaverse repos in github and don't see this function being used except in `admiral` everyone else uses `rlang::is_named()` https://github.com/search?q=org%3Apharmaverse+is_named&type=code

Reply:
ok @bms63 sounds good